### DISCUSSION TITLE
查询向量无反应

### DISCUSSION THREAD

**Discussion by lanceMsf** [milvus-io/milvus#17691] [Q&A and General discussion]
collection.search一直没有返回，也没有报错，是什么原因呢

**Comment by lanceMsf** [comment]
milvus服务日志链接: https://pan.baidu.com/s/1dxOskT0gLKQvpKk4SXloLg  密码: gr9n
--来自百度网盘超级会员V4的分享

**Comment by yhmo** [comment]
你的milvus版本是多少？
